odoo/tools/translate.py

```diff
@@ -147,7 +147,7 @@ def encode(s):
 TRANSLATED_ATTRS = {
     'string', 'help', 'sum', 'avg', 'confirm', 'placeholder', 'alt', 'title', 'aria-label',
     'aria-keyshortcuts', 'aria-placeholder', 'aria-roledescription', 'aria-valuetext',
-    'value_label',
+    'value_label', 'value',
```

Comment:
value is used in a lot of places for static content that must **not** be translated, please use one of the existing attribute

Reply:
I can not use one of the existing attribute, I need to translate the default value of an input...
What kind of trick should I use to make this work ?

Reply:
I guess we must indeed only translate value for input[type=text]. I guess we will have to define a new map of element -> attributes that should be translatable. Would that be complicated @fja-odoo ?

Reply:
In QWeb, you can do something like
```xml
<t t-set="input_val">My Value</t>
<input t-att-value="input_val" />
```
to have a translated attribute

Reply:
I do not think it is complex to do. Is it the right solution tho @mart-e ?

Reply:
@mart-e We obviously know that :) Here we are talking about editable contents, so without any computed qweb part in it. The user defines a form with default values for its page and he would like to translate those default values, like any other text content he enters.

Reply:
Sorry, I have to remind it often ;-) Ok I understand now. Should be possible to have it with a conditional map then. Probably [here](https://github.com/odoo/odoo/blob/426de6e901b77ecb0616706c9d07d85f0e9c589c/odoo/tools/translate.py#L272) (the input is not an inline element so not needed in the `if` above).
Will not work [for `attribute`](https://github.com/odoo/odoo/blob/426de6e901b77ecb0616706c9d07d85f0e9c589c/odoo/tools/translate.py#L219) in xpath though.

Reply:
Not sure if I get all of what the process function do but this is working for my use case. Also i think this will translate all input['text'] value attribute in the views. Is that a good or a bad thing ? looks like an improvement to me. [this](https://github.com/odoo/odoo/blob/426de6e901b77ecb0616706c9d07d85f0e9c589c/addons/mail/static/src/xml/discuss.xml#L242) and other will now be translated.

Reply:
> Also i think this will translate all input['text'] value attribute in the views.
Well that's what you meant to develop right? I am not sure to understand :laughing: I'll check tomorrow :+1: